hazelcast/test/src/CMakeLists.txt

```diff
@@ -53,7 +50,7 @@ IF (${CMAKE_SYSTEM_NAME} MATCHES "Windows")
     message(STATUS "Using THRIFT_INCLUDE_DIR: ${THRIFT_INCLUDE_DIR}")
     message(STATUS "Using THRIFT_LIBRARIES: ${THRIFT_LIBRARIES}")
     include_directories(${THRIFT_INCLUDE_DIR})
-    target_link_libraries(${HZ_EXE_NAME} PRIVATE ${THRIFT_LIBRARIES})
+    target_link_libraries(client_test PRIVATE ${THRIFT_LIBRARIES})
 
     IF (${HZ_LIB_TYPE} MATCHES "STATIC")
         message(STATUS "Shall use the STATIC Hazelcast library for the tests.")
```

Comment:
`DHAZELCAST_USE_STATIC` was being defined from the main `CMakeLists.txt` file. do we define again?